tests/unit/dataactvalidator/test_d3_detached_award_financial_assistance_1.py

```diff
@@ -0,0 +1,27 @@
+from tests.unit.dataactcore.factories.staging import DetachedAwardFinancialAssistanceFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'd3_detached_award_financial_assistance_1'
+
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "action_type", "record_type"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset == actual
+
+
+def test_success(database):
+    """ Tests success for when Action type is required for non-aggregate records (i.e., when RecordType = 2) """
+    det_award = DetachedAwardFinancialAssistanceFactory(record_type="2")
```

Comment:
Should there be a test here that proves that other record types (1) don't break it if they don't have action types?